gobblin-service/src/main/java/org/apache/gobblin/service/modules/orchestration/DagManagementTaskStreamImpl.java

```diff
@@ -146,7 +147,7 @@ public DagTask next() {
           }
         } catch (Exception e) {
           //TODO: need to handle exceptions gracefully
-          log.error("Exception getting DagAction from the queue / creating DagTask", e);
+          log.error("Exception getting DagAction from the queue or creating DagTask. dagAction - {}", dagAction == null ? "<null>" : dagAction, e);
```

Comment:
NBD, but:
```
log.error("Exception getting DagAction from the queue or creating DagTask. dagAction - {}", dagAction, e);
```
would, when `dagAction == null`, just result in:
```
Exception getting DagAction from the queue or creating DagTask. dagAction - null
```
which seems simpler to write and not much different than
```
Exception getting DagAction from the queue or creating DagTask. dagAction - <null>
```